SepulcherOfTheFirstOnes/Locales/frFR.lua

```diff
@@ -82,78 +82,78 @@ end
 end
 
 L = BigWigs:NewBossLocale("Lords of Dread", "frFR")
 if L then
-	--L.unto_darkness = "AoE Phase"
-	--L.cloud_of_carrion = "Carrion"
-	--L.empowered_cloud_of_carrion = "Big Carrion" -- Empowered Cloud of Carrion
-	--L.leeching_claws = "Frontal (M)"
-	--L.infiltration_of_dread = "Among Us"
-	--L.infiltration_removed = "Imposters found in %.1fs" -- "Imposters found in 1.1s" s = seconds
-	--L.fearful_trepidation = "Fears"
-	--L.slumber_cloud = "Clouds"
-	--L.anguishing_strike = "Frontal (K)"
-
-	--L.custom_on_repeating_biting_wound = "Repeating Biting Wound"
-	--L.custom_on_repeating_biting_wound_desc = "Repeating Biting Wound say messages with icons {rt7} to make it more visible."
+	L.unto_darkness = "Phase AoE"
+	L.cloud_of_carrion = "Nuée"
+	L.empowered_cloud_of_carrion = "Nuée ++" -- Empowered Cloud of Carrion
+	L.leeching_claws = "Devant le boss (M)"
+	L.infiltration_of_dread = "Ombre a trouvé"
+	L.infiltration_removed = "Ombre trouvé en %.1fs" -- "Imposters found in 1.1s" s = seconds
+	L.fearful_trepidation = "Peurs"
+	L.slumber_cloud = "Nuages"
+	L.anguishing_strike = "Devant le boss (K)"
+
+	L.custom_on_repeating_biting_wound = "Morsure Béante"
+	L.custom_on_repeating_biting_wound_desc = "Morsure Béante indique un message avec une icone {rt7} pour la rendre plus visible."
 end
 
 L = BigWigs:NewBossLocale("Rygelon", "frFR")
 if L then
-	--L.celestial_collapse = "Quasars"
-	--L.manifest_cosmos = "Cores"
-	--L.stellar_shroud = "Heal Absorb"
-	--L.knock = "Knock" -- Countdown knockbacking other players nearby. Knock 3, Knock 2, Knock 1
+	L.celestial_collapse = "Quasars"
+	L.manifest_cosmos = "Noyaux"
+	L.stellar_shroud = "Absorption des soins"
+	L.knock = "Repousse" -- Countdown knockbacking other players nearby. Knock 3, Knock 2, Knock 1
 end
 
 L = BigWigs:NewBossLocale("The Jailer", "frFR")
 if L then
-	--L.rune_of_damnation_countdown = "Countdown"
-	--L.rune_of_damnation_countdown_desc = "Countdown for the players who are affected by Rune of Damnation"
-	--L.jump = "Jump In"
-
-	--L.relentless_domination = "Domination"
-	--L.chains_of_oppression = "Pull Chains"
-	--L.unholy_attunement = "Pylons"
-	--L.shattering_blast = "Tank Blast"
-	--L.rune_of_compulsion = "Charms"
+	L.rune_of_damnation_countdown = "Compte à rebours"
+	L.rune_of_damnation_countdown_desc = "Compte à rebours pour les joueurs qui sont affecté par Rune de Damnation"
+	L.jump = "Plonger"
+
+	L.relentless_domination = "Domination"
+	L.chains_of_oppression = "Tirer les chaînes"
+	L.unholy_attunement = "Pylônes"
+	L.shattering_blast = "Exposion sur tank"
+	L.rune_of_compulsion = "Contrainte"
 	--L.desolation = "Azeroth Soak"
-	--L.chains_of_anguish = "Spread Chains"
-	--L.chain = "Chain"
-	--L.chain_target = "Chaining %s!"
-	--L.chains_remaining = "%d/%d Chains Broken"
-	--L.rune_of_domination = "Group Soaks"
+	L.chains_of_anguish = "Chaînes - Ecartez-vous"
```

Comment:
This seems to be describing a command to the player instead of describing the ability? Why?